site/en/docs/extensions/mv3/content_scripts/index.md

```diff
@@ -82,18 +82,19 @@ button.addEventListener("click", () =>
 With this change, both alerts appear in sequence when the button is clicked.
 
 {% Aside %}
-Not only does each extension run in its own isolated world, but content scripts
-and the web page do too. This means that none of these (web page, content
-scripts, and any running extensions) can access the context and variables of
-the others.
+
+Not only does each extension run in its own isolated world, but content scripts and the web page do
+too. This means that none of these (web page, content scripts, and any running extensions) can
+access the context and variables of the others.
+
 {% endAside %}
 
 {# youtube id="laLudeUmXHM" #}
 
 ## Inject scripts {: #functionality }
 
-Content scripts can be [declared statically][14]{% if false %}, [declared dynamically][32],
-{% endif %} or [programmatically injected][13].
+Content scripts can be [declared statically][14], [declared dynamically][32], or [programmatically
```

Comment:
for the new 32 and 33, should we just make them more legible names?  (Since this page already uses both?)

Reply:
Sure thing. I don't want to update all numbered links in this PR in order to limit scope creep, but I'd like to come back to this in another cleanup pass later.